pandas/core/algorithms.py

```diff
@@ -1092,18 +1093,19 @@ def checked_add_with_arr(
     # it is negative, we then check whether its sum with the element in
     # 'arr' exceeds np.iinfo(np.int64).min. If so, we have an overflow
     # error as well.
+    i8max = Timestamp.max.value  # GH#?
```

Comment:
can u make these consistent across all of the files

iow this is called different names but the same even in this change set

Reply:
will standardize in next commit.  this kludge really shouldn't be necessary

Reply:
ok ping when ready.

Reply:
ping, will be nice to finally get back to green

Reply:
right what i mean was can you standardize these names (you mostly do, but some are CAPITAL and some not). prefer to do it in this PR as going to backport.

Reply:
the ones in .py files are all `i8max`, `u8max`, or `iNaT`